src/main/resources/product_profile_registry.yaml

```diff
@@ -1,22 +1,88 @@
 name: RHELProducts
 productIds:
-  - 68   # Red Hat Enterprise Linux Desktop
-  - 69   # Red Hat Enterprise Linux Server
-  - 71   # Red Hat Enterprise Linux Workstation
-  - 72   # Red Hat Enterprise Linux for IBM z Systems
-  - 74   # Red Hat Enterprise Linux for Power, big endian
-  - 76   # Red Hat Enterprise Linux for Scientific Computing
-  - 279  # Red Hat Enterprise Linux for Power, little endian
-  - 290  # Red Hat OpenShift Container Platform
-  - 419  # Red Hat Enterprise Linux for ARM 64
-  - 420  # Red Hat Enterprise Linux for Power 9
-  - 479  # Red Hat Enterprise Linux for x86_64 (RHEL8)
+  - id: 68 # Red Hat Enterprise Linux Desktop
+    products:
+      - RHEL
+      - RHEL for x86
+      - RHEL Desktop
```

Comment:
Originally, I was thinking:

```yaml
68: # Red Hat Enterprise Linux Desktop
  - RHEL
  - RHEL for x86
  - RHEL Desktop
```

but now that I'm seeing this, I dig how the fields are explicit (rather than having an Object with arbitrary key/value types).

:+1: :100:

Reply:
I went that way to begin with, but getting SnakeYaml to deserialize that would have required a lot of customization.  (I tried doing that customization too but wasn't successful)